app/static/pages/home/situation_description/volunteer_situation.html

```diff
@@ -0,0 +1,12 @@
+<p class="oppia-foundation-bold-text">
+    Ideal for...
```

Comment:
Deindent by 2, ditto elsewhere.

Reply:
Done here and the two other HTML templates.